src/components/calculator.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+
+class Calculator extends Component {
+  render() {
+    // reference as a js object
+    const createDigits = () => {
+      const digits = [];
+
+      for (let i = 1; i < 10; i += 1) {
+        digits.push(
+          <button
+            type="button"
+            key={i}
+          >
+            {i}
+          </button>,
+        );
+      }
+      return digits;
+    };
+
+    return (
+      <div className="calculator">
+        <div className="display">
+          0
+        </div>
+
+        <div className="simpleOperators">
+          <button type="button">AC</button>
+          <button type="button">+/-</button>
+          <button type="button">%</button>
+        </div>
+        <div className="operators">
+          <button type="button">/</button>
+          <button type="button">*</button>
+          <button type="button">+</button>
+          <button type="button">-</button>
+          <button type="button">=</button>
+        </div>
+
+        <div className="digits">
+          { createDigits() }
+          <button type="button">0</button>
+          <button type="button">.</button>
+        </div>
```

Comment:
- [ ] Let's make sure you are following the given design. For now you are using symbols different from the one given on the design. Also check on colors.
- your design:
![your_design](https://user-images.githubusercontent.com/65860678/182591586-7ddfc3df-b5d8-4e02-a67b-3898a9aefffc.png)
- given design:

![Screenshot from 2022-08-03 12-48-50](https://user-images.githubusercontent.com/65860678/182591668-4866f688-e93a-4c49-8a27-afb63d73c8a0.png)